.changeset/hot-stingrays-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sumup/icons": minor
+---
+
+Update top 8 most used icons with new optimised versions.
```

Comment:
Remember to use past tense in changesets. Changesets should be precise in describing the change, explain why the change was made, and outline what users need to do to adopt the change (not applicable in this case).

```suggestion
Optimized the `Add`, `AddEmployees`, `AddItems`, `ArrowLeft`, `Calendar`, `Close`, `Edit`, and `Plus` icons to look more balanced when used alongside text.
```